tailwind.config.js

```diff
@@ -112,6 +112,7 @@ module.exports = {
       inter: ['Inter', 'sans-serif'],
     },
     fontSize: {
+      '2xs': ['.5rem', 1.5],
```

Comment:
If we are adding this we should update the other uses of `text-[0.5rem]` and `text-[.5rem]` to use this. (Maybe at a later point in a separate PR). It is also used in frequently enough that it would be fine to just use this as an arbitrary value here instead of adding it to the config.